examples/Textual-Responses.ipynb

```diff
@@ -0,0 +1,478 @@
+      ],
+      "metadata": {
+        "id": "PsMXwYO1ZCK1"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "We need to determine the encoder - a callable that takes a text and returns a NumPy array containing the corresponding embedding. For that, we will use the model from sentence_transformers. The model.encode returns the embedding."
+      ],
+      "metadata": {
+        "id": "Hk8bOkkUceaV"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "from sentence_transformers import SentenceTransformer\n",
+        "\n",
+        "model = SentenceTransformer('all-mpnet-base-v2')\n",
+        "\n",
+        "result_TextHRRASA = TextHRRASA(encoder = model.encode).fit_predict(df)"
+      ],
+      "metadata": {
+        "id": "ZBbAMGrnZCqK"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "KVeqVuTPBZaZ"
+      },
+      "source": [
+        " Let's compare the results of these methods - we will use word error rate (WER) from jiwer. It is a common metric of the performance of an automatic speech recognition system. This value indicates the percentage of words that were incorrectly predicted. The lower the value, the better the performance of the ASR system with a WER of 0 being a perfect score. Word error rate can then be computed as:\n",
+        "\\begin{align}\n",
+        "WER = \\frac{S+D+I}{S+D+C},\n",
+        "\\end{align}\n",
+        "Where: \n",
+        "- S is the number of substitutions, is the number of deletions, \n",
+        "- D is the number of deletions,\n",
+        "- I is the number of insertions, \n",
+        "- C is the number of correct words.\n",
+        "\n",
+        "\n",
+        "Let's see the difference!"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 14,
+      "metadata": {
+        "id": "3HnE7fdYBZFi",
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "outputId": "0bfc663d-9443-495e-bade-5df1caf5be2a"
+      },
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "The WER of TextRASA - 0.0907894234804838\n",
+            "The WER of TextHRRASA - 0.09415696009165553\n",
+            "The WER of ROVER - 0.07017374889239897\n"
+          ]
+        }
+      ],
+      "source": [
+        "from jiwer import wer\n",
+        "\n",
+        "input_TextRASA = list()\n",
```

Comment:
Please do not use a for loop here and just run `wer` on two pandas series values